packages/lab/src/form-field-next/FormFieldHelperText.css

```diff
@@ -1,4 +1,16 @@
-.saltFormFieldHelperText.saltText {
-  font-style: var(--saltFormField-helperText-fontStyle, var(--salt-editable-help-fontStyle));
+.saltFormFieldHelperText {
+  display: flex;
+  flex-direction: row;
   width: 100%;
 }
+
+.saltFormFieldHelperText .saltText {
+  font-style: var(--saltFormField-helperText-fontStyle, var(--salt-editable-help-fontStyle));
+}
+
+.saltFormFieldHelperText-withValidation {
+  column-gap: var(--salt-size-adornmentGap);
+}
+.saltFormFieldNext:has(.saltInputNext-disabled, .saltInputNext-readOnly) .saltStatusIndicator {
```

Comment:
@alycrys any ideas of how else to handle this for firefox?

Reply:
As discussed, I believe we can avoid using `:has` selector here

Reply:
i think you may have reviewed outdated code :)